inc/config.php

```diff
@@ -45,26 +45,6 @@
 Session::setPath();
 Session::start();
 
-//init cache
-$GLPI_CACHE = false;
```

Comment:
I think it is better to keep default value here

Reply:
Changed ;)